_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure istio ingress with an external load balancer NLB on AWS
+publish_date: April 08, 2018
+subtitle: Ingress AWS NLB
+attribution: Julien SENON
+
+order: 94
```

Comment:
I'm not sure if this order is correct or incorrect - I typically limit msyelf to reviewing _docs dir.  It does seem to come after the last blog post in the repo - will need someone more experienced in _blogs to weigh in.

Reply:
You right seems the opposite